tasks.py

```diff
@@ -0,0 +1,60 @@
+import csv
+import time
+from urllib.parse import urlparse
+from robocorp.tasks import task
+from robocorp import browser
+
+
+@task
+def take_website_screenshots():
+    """Take screenshots of all websites found in websites.csv file."""
+    with open("websites.csv", mode="r") as csv_file:
+        csv_reader = csv.DictReader(csv_file)
+        websites = list(csv_reader)
+
+    for website in websites:
+        browser.goto(website["url"])
+        page = browser.page()
+
+        accept_cookies_and_consents(website)
+
+        # some websites have animations on cookies and consents so wait for that to disappear
+        time.sleep(1)
```

Comment:
Makes sense, but considering only one page has this behaviour from the list, I think it is easier for people to follow with a time.sleep() rather than introducing another selector / another column in the csv.